Comment DELETE operation (#14)
AMWA-TV/nmos-sink-metadata-processing: docs/1.2. Behaviour.md

## Senders

> `/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender, let the underlying component choose exactly which format to use at the moment and dictate that format changes which are not consequent to Media Profiles change MUST lay within Media Profiles set. A client can also pass a single Media Profile to specify the exact format.

Reviewer: This is from #13 ?

Author: Yes. I thought it's just extra to #13 but it may be redundant.

Author: Moved this out.

> `/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender, let the underlying component choose exactly which format to use at the moment and dictate that format changes which are not consequent to Media Profiles change MUST lay within Media Profiles set. A client can also pass a single Media Profile to specify the exact format.

Author: ```suggestion
`/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the underlying component control which it choose exactly which format to use. A client can also pass a single Media Profile to specify the exact format.
```
